jee-learning-platform/backend/app/routes/content.py

```
@@ -0,0 +1,318 @@
+    return db_chapter
+
+# Topic Routes
+@router.get("/topics/{topic_id}", response_model=schemas.TopicContent)
+async def get_topic_content(
+    topic_id: int,
+    current_user: models.User = Depends(get_current_active_user),
+    db: Session = Depends(get_db)
+):
+    """Get complete topic content including blocks and checkpoints."""
+    # Start session analytics
+    session = AnalyticsService.start_session(db, current_user.id, topic_id)
```

Ruff (0.11.9)

Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)